package.json

```diff
@@ -30,13 +30,21 @@
     "twitter": "1.7.1"
   },
   "devDependencies": {
+    "commitizen": "4.1.2",
+    "cz-conventional-changelog": "^3.2.0",
     "husky": "4.2.5",
     "prettier": "2.0.5",
     "pretty-quick": "2.0.1"
   },
   "husky": {
     "hooks": {
-      "pre-commit": "pretty-quick --staged --verbose"
+      "pre-commit": "pretty-quick --staged --verbose",
+      "prepare-commit-msg": "exec < /dev/tty && git cz --hook || true"
```

Comment:
I would recommend using [Husky](https://github.com/typicode/husky) which made easy to manage the git-hooks and [Commitlint](https://github.com/conventional-changelog/commitlint) to make sure that all commits pushed have the same pattern.

Reply:
I'm using `husky` as suggested by `commitizen`. Do you suggest a different setup?

Gonna check commitling, thanks 😁

Reply:
Sorry, I just thought that these scripts were in "scripts" instead of "husky.hooks", my bad. 
But it seems ok, only missing the committing so :)